react/content-list.jsx

```diff
@@ -17,11 +17,18 @@ let ContentList = React.createClass({
     }
   },
 
-  setStateFromDattCore: function () {
+  setStateFromDattCore: function (skipSetup) {
```

Comment:
As mentioned previously, IMO, the "setup" simply shouldn't exist at all. I don't think we gain anything by asking the user to "setup" their account. It removes one of the great usability benefits of a decentralized system: zero setup.